package.json

```diff
@@ -50,5 +50,8 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "styled-jsx": "^3.2.1"
-  }
+  },
+  "files": [
+    "./build"
```

Comment:
If we expose non-js files directly in, i.e., a `./css` folder we could add that folder to the `files` array and, if we want to get fancy, enforce that removing a file from a non-build published folder is automatically a breaking change.